feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts

```diff
@@ -60,12 +61,38 @@ export class ConfiguratorTabBarComponent {
       )
     );
 
+  /**
+   * @deprecated Use getPageType$ and isOverviewPage(ConfiguratorRouter.PageType)
+   * instead
+   */
   isOverviewPage$: Observable<boolean> = this.routerData$.pipe(
     map(
       (routerData) =>
         routerData.pageType === ConfiguratorRouter.PageType.OVERVIEW
     )
   );
+  /**
+   * Retrieves current page type
```

Comment:
Please add a dot at the end of the explanation, then add an empty line between an explanation and a  return value.